dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -101,6 +101,21 @@ def __add__(self, other):
                 self._profiles[profile_name] + other._profiles[profile_name]
             )
         return merged_profile_compiler
+    
+    def diff(self, other, options=None):
+        """
+        Finds the difference between 2 compilers and returns the report
+
+        :param other: profile compiler finding the difference with this one.
+        :type other: BaseCompiler
+        :return: difference of the profiles
+        :rtype: dict
+        """
+        if type(other) is not type(self):
+            raise TypeError('`{}` and `{}` are not of the same profile compiler'
+                            ' type.'.format(type(self).__name__,
+                                            type(other).__name__))
+        return {}
```

Comment:
I find it strange to return an empty dict, but we don't use it.

Reply:
Good point, I use it now